src/main/java/com/cburch/logisim/fpga/hdlgenerator/CircuitHDLGeneratorFactory.java

```diff
@@ -281,8 +281,8 @@ public String getComponentStringIdentifier() {
             }
             while (OneLine.length() < SallignmentSize) OneLine.append(" ");
 
-            Contents.add("   {{assign}} {{1}} {{=}} {{2}}{{3}}{{bracketOpen}}{{4}}{{bracketClose}};",
-                OneLine, BusName, TheNets.getNetId(Source.getParentNet()), Source.getParentNetBitIndex());
+            Contents.addUnique(LineBuffer.format("   {{assign}} {{1}} {{=}} {{2}}{{3}}{{bracketOpen}}{{4}}{{bracketClose}};",
```

Comment:
As I said, if you merge #960 first, then explicit call to `LineBuffer.format()` is no longer needed as there's `addUnique(String fmt, Object... args)` implementation already that will route args thru `format()` prior adding. Same below.